[CI] GoogleCloudStorageBlobContainerStatsTests testResumableWrite failing

**Build Scans:**
- [elasticsearch-periodic #6950 / concurrent-search-tests](https://gradle-enterprise.elastic.co/s/x4u2hsr5w25he)
- [elasticsearch-pull-request #62095 / part-1](https://gradle-enterprise.elastic.co/s/faesqncusq2jk)
- [elasticsearch-pull-request #62094 / part-1](https://gradle-enterprise.elastic.co/s/esnrml4sasfe4)
- [elasticsearch-pull-request #62049 / part-1](https://gradle-enterprise.elastic.co/s/wqwwvv6ltrbac)

**Reproduction Line:**
```
./gradlew ":modules:repository-gcs:test" --tests "org.elasticsearch.repositories.gcs.GoogleCloudStorageBlobContainerStatsTests.testResumableWrite" -Dtests.seed=E949297FEE50A8CC -Dtests.jvm.argline="-Des.concurrent_search=true" -Dtests.locale=ku-Latn-TR -Dtests.timezone=SystemV/CST6CDT -Druntime.java=23
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.repositories.gcs.GoogleCloudStorageBlobContainerStatsTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testResumableWrite),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: null
```

**Issue Reasons:**
- [main] 4 failures in test testResumableWrite (1.8% fail rate in 224 executions)
- [main] 3 failures in step part-1 (3.0% fail rate in 101 executions)
- [main] 3 failures in pipeline elasticsearch-pull-request (3.0% fail rate in 100 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/modules/repository-gcs/src/test/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobContainerStatsTests.java
+++ b/modules/repository-gcs/src/test/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobContainerStatsTests.java
@@ -158,7 +158,7 @@ public class GoogleCloudStorageBlobContainerStatsTests extends ESTestCase {
 
         final OperationPurpose purpose = randomPurpose();
         final String blobName = randomIdentifier();
-        final int parts = between(1, 10);
+        final int parts = between(2, 10);
         final int maxPartSize = GoogleCloudStorageBlobStore.SDK_DEFAULT_CHUNK_SIZE;
         final int size = (parts - 1) * maxPartSize + between(1, maxPartSize);
         assert size >= store.getLargeBlobThresholdInBytes();

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -375,9 +375,6 @@ tests:
 - class: org.elasticsearch.lucene.spatial.CartesianCentroidCalculatorTests
   method: testAddDifferentDimensionalType
   issue: https://github.com/elastic/elasticsearch/issues/124609
-- class: org.elasticsearch.repositories.gcs.GoogleCloudStorageBlobContainerStatsTests
-  method: testResumableWrite
-  issue: https://github.com/elastic/elasticsearch/issues/124648
 
 # Examples:
 #

